### PR TITLE
update links in README.md

### PR DESCRIPTION
**Pull request by 0xallan**
Link to **_nitro contract_** addresses was broken (https://developer.arbitrum.io/useful-addresses)

Separate link to **_token bridge contract_** addresses could be replaced with the same updated docs link (https://docs.arbitrum.io/for-devs/dev-tools-and-resources/chain-info)

### CODE REVIEW

**Comment by hkalodner** [comment]
@cla-bot check

File: README.md [hunk 1]
```diff
@@ -5,9 +5,7 @@ It includes the rollup and fraud proof smart contracts, as well as interfaces fo
 
 For more information see https://developer.arbitrum.io/intro
 
-For the deployed addresses of these contracts for Arbitrum chains see https://developer.arbitrum.io/useful-addresses
-
-For the token bridge contracts see https://github.com/OffchainLabs/token-bridge-contracts
```
Comment (hkalodner) [inline README.md:10]: I think we might want to leave this line since it's helpful to link people to that other repo for discovery

Comment (0xallan) [reply README.md:10]: reinstated the token bridge contracts repo link!

**Review by joshuacolvin0 — DISMISSED** [review]
LGTM

**Review by gzeoneth — APPROVED** [review]

**Pull request merged**